wiki/Import-export guide for ContentPages with linked WhatsAppTemplates.md

```diff
@@ -0,0 +1,336 @@
+| `page_id` | No | Internal page ID | `101` |
+| `slug` | Yes | Unique identifier for the page | `pregnancy-week-1` |
+| `parent` | No | Parent page title (for organizing in tree) | `Pregnancy Journey` |
+| `web_title` | No | Title for web display | `Week 1: First Steps` |
+| `web_subtitle` | No | Subtitle for web display | `Your pregnancy begins` |
+| `web_body` | No | Content for web channel | `Welcome to week 1!` |
+| `whatsapp_title` | No | Title for WhatsApp | `Week 1` |
+| `whatsapp_body` | No | Message text for WhatsApp | `Welcome to week 1!` |
+| `whatsapp_template_slug` | No | Reference to a WhatsAppTemplate | `appointment-reminder` |
+| `variation_title` | No | Title for message variations | |
+| `variation_body` | No | Body for message variations | |
+| `list_title` | No | Title for list messages | |
+| `list_items` | No | Items in list messages (JSON format) | |
+| `sms_title` | No | Title for SMS channel | |
+| `sms_body` | No | Message text for SMS | |
+| `ussd_title` | No | Title for USSD channel | |
+| `ussd_body` | No | Message text for USSD | |
+| `messenger_title` | No | Title for Facebook Messenger | |
+| `messenger_body` | No | Message text for Messenger | |
+| `viber_title` | No | Title for Viber | |
+| `viber_body` | No | Message text for Viber | |
+| `translation_tag` | No | Tag for translation management | |
+| `tags` | No | Categories/tags (comma-separated) | `pregnancy,health` |
+| `quick_replies` | No | Quick reply buttons (JSON format) | `[{"type": "next_message", "title": "Next"}]` |
+| `triggers` | No | Custom triggers (JSON format) | |
+| `buttons` | No | Interactive buttons (JSON format) | |
+| `image_link` | No | Image for the page | |
+| `doc_link` | No | Document attachment | |
+| `media_link` | No | Media file (audio/video) | |
+| `related_pages` | No | Related pages (comma-separated slugs) | `week-2,week-3` |
+| `footer` | No | Footer text for messages | |
+| `language_code` | Yes | Language code (or use `locale`) | `en` |
+
+### ContentPage CSV Example
+
+```csv
+structure,message,page_id,slug,parent,web_title,web_subtitle,web_body,whatsapp_title,whatsapp_body,whatsapp_template_slug,locale,tags,quick_replies
+Menu 1,0,100,main-menu,,Main Menu,,,,,en,,
+Sub 1.1,1,101,pregnancy-info,Main Menu,Pregnancy Info,,,Pregnancy Info,Get helpful info about your pregnancy journey.,,en,pregnancy,"[{""type"": ""next_message"", ""title"": ""Next""}]"
+Sub 1.2,1,102,appointment,Main Menu,Appointments,,,Appointment,,appointment-reminder,en,appointments,
+```
+
+### Importing ContentPages
+
+**Via Admin UI:**
+1. Navigate to `/admin/import/`
+2. Upload CSV/Excel file
+3. Select purge option
+4. Select locale filter (optional)
+5. Click Import
+
+**Import Parameters:**
+- **Purge**: `True` deletes ALL existing pages before import. **Warning**: This removes entire content tree.
+- **Locale**: Filter to specific locale
+
+### Exporting ContentPages
+
+**Via Admin UI:**
+1. Go to Pages list in admin
```

Comment:
I think it's `ContentPages` in the UI. The `Pages` give you the tree view, and you can't export from there (which is confusing)